app/main.py

```diff
@@ -1,3 +1,43 @@
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Union[int, float, "Distance"]) -> object:
```

Comment:
Change your type annotation to use the latest syntaxis. Besides that, be more specific in the return type annotation, in Python everything is an object:
```suggestion
    def __add__(self, other: int | float | Distance) -> Distance:
```
Apply this change to all places where applicable. In case you face incompatibility errors, use `from __future__ import annotation`.